### PR TITLE
Cyborg<->Human Combat Rebalance: Cyborg Damage Slowdown, Stamina Damage, and Stun Batons

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Cyborgs are no longer hardstunned by Flashes, and Engineering Borgs now have a normal stun baton instead of a stun arm. Cyborgs are now blinded and blurred by flashes, as it overloads their visual sensors.
Cyborgs now have damage slowdown.
Cyborgs now take Stamina Damage, and can enter Stamina Crit.
@Kylerace approved as a Maintainer.
Discord_l6DikHDeeT

## Why It's Good For The Game
Hardstun bad because hardstun is an instant shutdown of combat and an instant win state.
Both of these mechanics were designed back in the dark ages when Security had Tasers and Stun Batons were a one click instant win.

The new stun baton will now provide people at least a slight chance to escape or fight back.
The new flash on cyborg effect will now provide cyborgs at least a slight chance to escape or fight back.

Movement Speed being constant on cyborgs has been a passive pain in the ass for balancing as cyborgs are either operating at full combat effectiveness or dead, with very little inbetween. Damage slowdown also makes hits and damage feel impactful naturally.

Being able to non-lethally take down cyborgs is important, so now you can stun baton cyborgs. Flashes will also do 60 stamina damage, allowing 2 consecutive flashes to stamina-crit a cyborg.

## Changelog
:cl:
balance: Cyborgs are now blinded and blurred by flashes, as it overloads their visual sensors.
balance: Cyborgs are no longer hard stunned by flashes.
balance: Engineering Cyborgs when emagged/malf AI'd no longer have Stun Arms.
balance: Engineering Cyborgs when emagged/malf AI'd now have Stun Batons.
balance: Cyborgs are now effected by movement slowdown from damage.
balance: Cyborgs can now take Stamina Damage, and can enter stamina crit.
balance: Flashes now do stamina damage to cyborgs.
balance: You can now Stun Baton/Cattle Prod cyborgs.
/:cl:

### CODE REVIEW

**Review by Kylerace — APPROVED** [review]
exact balance can be debated over but im completely for moving hard stuns to stuns.

**Comment by carshalash** [comment]
After this gets merged can we finally nerf the cult stun?

**Comment by Kylerace** [comment]
> 
> 
> After this gets merged can we finally nerf the cult stun?

as long as im convinced cult is ok with whatever you change it to then im all for it

**Comment by Mothblocks** [comment]
do not remove the cult stun, it is the only way cult has any actual chance

**Comment by Mothblocks** [comment]
this sucks by the way as long as borgs have no damage slowdown

**Comment by cacogen** [comment]
>Hardstun bad because hardstun is an instant shutdown of combat and an instant win state

You have to actually maintain the stun long enough to win which isn't always possible given the chaos combat can sometimes be. But being flashed as a cyborg always feels like a huge fuck you and the stun arm sucks to be the victim of so I am ok with this.

**Comment by Mothblocks** [comment]
i dont think you guys realize how frustrating it is to fight a borg without a flash

**Comment by Kylerace** [comment]
i can take it without the flash changes

**Comment by LemonInTheDark** [comment]
Please actually argue for your changes. These things were intentional, and exist or existed for a reason. Just using hardstun as a dirty word is silly at this point.

**Comment by Iamgoofball** [comment]
Hardstun is a dirty word.

**Comment by tralezab** [comment]
image

this was what I said on it, I like the uniqueness to the gameplay of silicons and i think that uniqueness shouldn't come from having hard stuns and ignoring a large amount of effects from combat.

but that being said: it's best you make a design for how you're going to shift borgs in a new direction so they stay fresh and aren't just like the collective napkin thoughts of everyone who passes by and notices they get hardstunned

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Until borgs have damage slowdown, this is a bad PR. 

Furthermore, why not make the flash act like a stun baton where it slows you for a bit THEN stuns, rather than just making it useless?

**Comment by LemonInTheDark** [comment]
> 
> 
> Hardstun is a dirty word.

Ha*dstun is not a dirty word

**Comment by ATH1909** [comment]
> this sucks by the way as long as borgs have no damage slowdown

IMO, the flash should apply a heavy slowdown effect, but borgs shouldn't take normal damage slowdown

**Comment by carshalash** [comment]
> What if we just remove silicons

image

**Comment by Krysonism** [comment]
Makes stamina weapons even more of the solution to every problem.

I think we should be encouraging diversity of combat strategies, not make the batong and disabler the only game in town.

**Comment by SuperNovaa41** [comment]
> Don't forget about the Ai trying to stop borgs from disabling their power, if the mounted flashes don't hardstun borgs in their chamber they'll need another way to stop rogue borgs.

Laser Panel > Attack Cyborgs? On.

**Comment by SuperNovaa41** [comment]
I feel like if you atomized this, you could've gotten a lot more of what you wanted

**Comment by EricZilla** [comment]
I really dont think borgs should have stamina, or at least not be able to be stamcrit until they lose all their modules or something. Its a machine, its only gonna to falter if you REALLY hurt it. How in the world does it even HAVE stamina

**Comment by Iamgoofball** [comment]
Mining Cyborgs need a module rework entirely

**Comment by Iamgoofball** [comment]
I'll pull the baton changes as a maintainer requested I pull those.

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
Laserpointers also hardstun borgs
Same with flashbangs and mounted flashers, they paralyze borgs

If you're doing this change, you should also touch on those items

**Comment by itseasytosee** [comment]
@Wallemations humans are hard stunned by those things as well.

**Review by Kylerace — CHANGES_REQUESTED** [review]

**Comment by Coastfront** [comment]
Oh no my computer got tired guys.
In other news, This feels like a pretty hard dick slap to borgs, And a pretty hard dick elongation for sec that now there is yet another enemy and antagonist that they can just batong instead of having to use literally any other tool that sec has. Borgs having damage slowdown makes sense, Because it could be justified by just their wheels getting jammed/their robo leg joints getting jammed. But baton stuns? really? relegate the flash in carbon vs cyborg combat to carbons have minimal reason to use it if its just a blind+blur, baton reigns supreme because batong. This also leads to some other things such as: (unless if someone has already mentioned these), iirc boxing gloves still deal stamina damage, would this mean that boxing a cyborg into stam crit can work. Does this now mean that the glorified beating stick that heads of staff get is now capable of somehow making a computer tired, also makes cyborgs who, Have almost, if not no ranged combat capabilities, omega fucked against disablers since they make "damage" slowdown more of a problem since iirc they're faster to slowdown than lasers are. why must the fucking net launcher make robot tired. w h y.

**Comment by Stonetear** [comment]
> balance: Cyborgs are now effected by movement slowdown from damage.
Please no.  I'm not even a silicon player and I think this is a horrible change.  Borgs are already not very good at combat.  One of their most useful abilities is to evacuate (themselves or others) faster than pretty much anything else.

**Comment by optimumtact** [comment]
I need to see a coherent design document on a change as big as this.

**Comment by Iamgoofball** [comment]
👍

**Pull request closed without merging**
